saleor/dashboard/discount/forms.py

```diff
@@ -43,6 +59,28 @@ class VoucherForm(forms.ModelForm):
     class Meta:
         model = Voucher
         exclude = ['limit', 'apply_to', 'product', 'category']
+        labels = {
+            'type': pgettext_lazy(
+                'Discount type',
+                'Discount type'),
+            'name': pgettext_lazy(
+                'Voucher name',
+                'Name'),
+            'code': pgettext_lazy(
+                'Voucher code',
```

Comment:
"Coupon code" as context would make it clear we're not dealing with source code here.